plugin.php

```diff
@@ -957,7 +957,7 @@ public function render_tools_page() {
 			<div class='notice notice-warning inline'>
 				<p>
 						<?php
-						echo '<strong>' . esc_html__( 'Important', 'updates-api-inspector' ) . '</strong>:' .
+						echo '<strong>' . esc_html__( 'Important', 'updates-api-inspector' ) . '</strong>:&nbsp;' .
```

Comment:
Can you please break the 2 "string" changes out into a separate PR...so that this one is just about `pretty_print()`?